app-web/gitea/autobuild/build

```diff
@@ -0,0 +1,16 @@
+abinfo "Linking config directory ..."
+LDFLAGS="-X 'code.gitea.io/gitea/modules/setting.CustomPath=/etc/gitea' \
+    -X 'code.gitea.io/gitea/modules/setting.CustomConf=app.ini' \
+    -X 'code.gitea.io/gitea/modules/setting.AppWorkPath=/var/lib/gitea/'"
+
+abinfo "Building gitea ..."
+TAGS="bindata sqlite sqlite_unlock_notify pam" make build
+ 
+abinfo "Installing gitea ..."
+install -Dvm755 "$SRCDIR"/gitea "$PKGDIR"/usr/bin/gitea
+
+abinfo "Creating work-dir and config-dir ..."
+mkdir -pv "$PKGDIR"/var/lib/gitea/{custom,data,log}
+chmod -R 750 -v "$PKGDIR"/var/lib/gitea/
+mkdir -pv "$PKGDIR"/etc/gitea
+chmod 770 -v "$PKGDIR"/etc/gitea
```

Comment:
You can use `install -dvm [PERM]` to create directories and modify the permission of the directories easily.

Reply:
Thanks, modified.